core/install_scss.js

```diff
@@ -0,0 +1,10 @@
+var Download = require('download');
```

Comment:
Ahh it left out something else too, where it installs it, slightly important. I will make a new branch and request. Oi